src/accountsdb/db.zig

```diff
@@ -1313,6 +1362,8 @@ pub const AccountsDB = struct {
     /// as the data field ([]u8) for each account.
     /// Returns the unclean file id.
     pub fn flushSlot(self: *Self, slot: Slot) !FileId {
+        var timer = std.time.Timer.start() catch @panic("Timer unsupported");
+
```

Comment:
outside of scope for a comment but i think with these metric changes we can delete the result returns on clean and shrink in the `runManagerLoop` and just return void instead (eg, change from `const clean_result = try self.cleanAccountFiles(` to `try self.cleanAccountFiles(` and delete `self.logger.debugf("clean_result: {any}", .{clean_result});` for both clean and shrink)

Reply:
I'm thinking to make this change after removing the error union from Timer.start(). Happy to "resolve" this for now and pick it up later?